mobile-browser-based-version/src/components/TaskForm.vue

```diff
@@ -0,0 +1,1051 @@
+              .number()
+              .integer()
+              .positive()
+              .required(),
+            as: "input",
+            type: "number",
+            default: "eg. : 1",
+          },
+          {
+            id: "IMAGE_H",
+            name: "Hight of Image (pixels)",
+            yup: yup
+              .number()
+              .integer()
+              .positive()
+              .required(),
+            as: "input",
+            type: "number",
+            default: "eg. : 28",
+          },
+          {
+            id: "IMAGE_W",
+            name: "Width of Image (pixels)",
+            yup: yup
+              .number()
+              .integer()
+              .positive()
+              .required(),
+            as: "input",
+            type: "number",
+            default: "eg. : 28",
+          },
+          {
+            id: "LABEL_LIST",
+            name: "List of labels",
+            yup: yup
+              .array()
+              .of(yup.string())
+              .min(1)
+              .required(),
+            as: "input",
+            type: "array",
+            default: "eg. : 0",
+          },
+          {
+            id: "NUM_CLASSES",
+            name: "Number of classes",
+            yup: yup
+              .number()
+              .positive()
+              .required(),
+            as: "input",
+            type: "number",
+            default: "eg. : 2",
+          },
+          {
+            id: "LABEL_ASSIGNMENT",
+            name: "List of labels",
+            yup: yup
+              .array()
+              .of(
+                yup
+                  .object()
+                  .shape({
+                    stringLabel: yup
+                      .string()
+                      .required()
+                      .label("Label (string)"),
+                    intLabel: yup
+                      .string()
+                      .required()
+                      .label("Label (int)"),
+                  })
+                  .required()
+              )
+              .strict(),
+            as: "input",
+            type: "arrayObject",
+            elements: [
+              {
+                key: "stringLabel",
+                default: "eg. : airplane",
+              },
+              {
+                key: "intLabel",
+                default: "eg. : 1",
+              },
+            ],
+          },
+          {
+            id: "csvLabels",
+            name: "Are labels stored as CSV ?",
+            yup: yup.boolean().required(),
+            as: "input",
+            type: "checkbox",
+            default: false,
+          },
+          {
+            id: "aggregateImagesById",
+            name: "Aggregate Images By Id",
+            yup: yup.boolean(),
+            as: "input",
+            type: "checkbox",
+            default: false,
+          },
+        ],
+        other: [],
+      },
+      // *** Section ***
+      {
+        title: "Model Compile Data",
```

Comment:
i don't understand the meaning of this one

Reply:
I just copy-pasted the fields that already are in the `server/tasks.json` file

Reply:
I guess they should be used by a component of the platform at some point but I have to say I haven't checked